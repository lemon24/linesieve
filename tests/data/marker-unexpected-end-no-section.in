--section '(\S+):$' --success ok --failure fail show $nothing
---
aaa
