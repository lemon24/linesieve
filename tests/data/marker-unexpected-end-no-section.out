.
aaa
linesieve: unexpected end during <no-section>
